publication-event-handlers/src/test/java/no/unit/nva/publication/events/handlers/tickets/PendingPublishingRequestEventHandlerTest.java

```diff
@@ -1,58 +1,45 @@
 package no.unit.nva.publication.events.handlers.tickets;
 
-import static java.net.HttpURLConnection.HTTP_NOT_FOUND;
-import static java.net.HttpURLConnection.HTTP_OK;
-import static no.unit.nva.model.testing.PublicationGenerator.randomPublication;
-import static no.unit.nva.publication.events.handlers.tickets.PendingPublishingRequestEventHandler.BACKEND_CLIENT_SECRET_NAME;
-import static no.unit.nva.testutils.RandomDataGenerator.randomString;
-import static no.unit.nva.testutils.RandomDataGenerator.randomUri;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.core.Is.is;
-import static org.hamcrest.core.IsEqual.equalTo;
-import static org.hamcrest.core.StringContains.containsString;
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
-import java.io.ByteArrayOutputStream;
-import java.io.IOException;
-import java.io.InputStream;
-import java.net.URI;
-import java.nio.file.Path;
-import java.time.Clock;
 import no.unit.nva.commons.json.JsonUtils;
 import no.unit.nva.events.models.EventReference;
 import no.unit.nva.model.Organization;
 import no.unit.nva.model.Publication;
 import no.unit.nva.model.PublicationStatus;
 import no.unit.nva.model.associatedartifacts.file.PublishedFile;
 import no.unit.nva.publication.events.bodies.DataEntryUpdateEvent;
-import no.unit.nva.publication.model.BackendClientCredentials;
-import no.unit.nva.publication.model.business.Entity;
-import no.unit.nva.publication.model.business.PublishingRequestCase;
-import no.unit.nva.publication.model.business.Resource;
-import no.unit.nva.publication.model.business.TicketEntry;
-import no.unit.nva.publication.model.business.TicketStatus;
-import no.unit.nva.publication.model.business.UserInstance;
+import no.unit.nva.publication.model.business.*;
```

Comment:
Tror vi har noe unit style som sier at vi skal explisitt importere alt som trengs for optimaliserings årsaker. Spør Rurik

Reply:
Ser at det er årsaken til at codacy feiler.

Reply:
Ja, har problemer med Checkstyle, den vil ikke laste prosjektdefinisjonene vi har, så den tar google sin checkstyle, som forkorter importene, ganske irriterende, jeg så også dette etter at codacy feilet, aka det er fikset.